### PR TITLE
Update booking show page

### PR DESCRIPTION
**Pull request by JulianABodenschatz**
update booking show page 
– incl. confirmation banner, buttons and background photo
– excl. total price and cancel btn

### CODE REVIEW

File: app/views/shared/_booking_train_card.html.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="card-provider">
+      <img src="" alt="logo">
+      <small>
+        Deutsche Bahn
+        <br>
+        <%= @booking.package.train_from.train_number %>
+      </small>
+    </div>
+    <div class="card-price">
+      <p><%= @booking.package.train_from.price + @booking.package.train_to.price %>€ per Trip</p>
```
Comment (maguseva) [inline app/views/shared/_booking_train_card.html.erb]: it's now `package.train_price`

File: app/views/shared/_booking_train_card.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+              <i><%= @booking.package.train_from.category %></i>
+              <p><%= @booking.package.start_date.strftime('%b %d, %Y') %>, <%= @booking.package.train_from.dep_time %> – <%= @booking.package.train_from.arr_time %></p>
+            </small>
+          </span>
+        </div>
+        <div class="schedule hidden">
+          <span>
+            <abbr><%= @booking.package.train_to.dep_city %> – <%=@booking.package.train_to.arr_city%></abbr>
+            <small>
+              <i><%= @booking.package.train_from.category %></i>
```
Comment (maguseva) [inline app/views/shared/_booking_train_card.html.erb]: should it be `train_to.category`?

File: app/views/shared/_booking_train_card.html.erb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+              <p><%= @booking.package.start_date.strftime('%b %d, %Y') %>, <%= @booking.package.train_to.dep_time %> – <%= @booking.package.train_to.arr_time %></p>
```
Comment (maguseva) [inline app/views/shared/_booking_train_card.html.erb]: `package.end_date` ?

File: app/views/shared/_booking_train_card.html.erb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+          </span>
+        </div>
+      </div>
+    </div>
+    <div class="card-provider">
+      <img src="" alt="logo">
+      <small>
+        Deutsche Bahn
+        <br>
+        <%= @booking.package.train_from.train_number %>
```
Comment (maguseva) [inline app/views/shared/_booking_train_card.html.erb]: train_to?

File: app/views/shared/_booking_train_card.html.erb (hunk 1 above)
Comment (JulianABodenschatz) [reply app/views/shared/_booking_train_card.html.erb]: Ok

File: app/views/shared/_booking_train_card.html.erb (hunk 2 above)
Comment (JulianABodenschatz) [reply app/views/shared/_booking_train_card.html.erb]: We will keep it for now. Let's discuss it later;)

File: app/views/shared/_booking_train_card.html.erb (hunk 3 above)
Comment (JulianABodenschatz) [reply app/views/shared/_booking_train_card.html.erb]: Ok

File: app/views/shared/_booking_train_card.html.erb (hunk 4 above)
Comment (JulianABodenschatz) [reply app/views/shared/_booking_train_card.html.erb]: Ok

**Pull request merged**